Wiki-Style-Guide.md

```diff
@@ -1,19 +1,19 @@
-# What Does "Wiki" Means?
+# What Does “Wiki” Mean?
```

Comment:
I don't think we should change the quotes. They were standard quotes.

Symbol|Unicode character|Oct|Dec|Hex|HTML|ref|remark
:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:
“|left double quotation mark|020034|8220|0x201C|`&ldquo;`|http://unicodelookup.com/#“/1|non-standard
"|quotation mark|042|34|0x22|`&quot;`|http://unicodelookup.com/#"/1|standard in coding